data/campaigns/fri04.wmf/scripting/starting_conditions.lua

```diff
@@ -0,0 +1,298 @@
+         shovel = 6,
+         granite = 40,
+         log = 30,
+         water = 12,
+         wheat = 4,
+         wine = 8,
+         planks = 45,
+         wool = 2,
+   }
+   hq:set_workers {
+         empire_armorsmith = 1,
+         empire_brewer = 1,
+         empire_builder = 10,
+         empire_carrier = 40,
+         empire_charcoal_burner = 1,
+         empire_geologist = 4,
+         empire_lumberjack = 3,
+         empire_miner = 4,
+         empire_stonemason = 2,
+         empire_toolsmith = 2,
+         empire_trainer = 3,
+         empire_weaponsmith = 1,
+         empire_donkey = 5,
+   }
+   soldiers = {
+      {{1,2,0,1}, 70},
+      {{2,3,0,2}, 80},
+      {{4,4,0,2}, 90},
+   }
+   hq:set_soldiers(soldiers[difficulty][1], soldiers[difficulty][2])
+end
+
+-- =======================================================================
+--                                 Player 4
+-- =======================================================================
+p4.hidden_from_general_statistics = true
+p4:allow_buildings("all")
+p4:forbid_buildings {
+   "amazons_port",
+   "amazons_shipyard",
+}
+
+hq = p4:place_building("amazons_headquarters", map.player_slots[4].starting_field, false, true)
+hq:set_wares {
+         log = 100,
+         granite = 50,
+         rope = 30,
+         liana = 15,
+         ironwood = 15,
+         rubber = 15,
+         balsa = 9,
+         water = 20,
+         fish = 10,
+         meat = 10,
+         cassavaroot = 15,
+         ration = 5,
+         coal = 5,
+         gold = 4,
+         amazons_bread = 5,
+         chocolate = 6,
+         hammer = 10,
+         shovel = 5,
+         cocoa_beans = 3,
+         tunic = 5,
+         chisel = 3,
+         pick = 2,
+         machete = 2,
+         spear_wooden = 6,
+         stone_bowl = 2,
+         needles = 2,
+}
+hq:set_workers {
+         amazons_cook = 3,
+         amazons_builder = 10,
+         amazons_gold_digger = 1,
+         amazons_gold_smelter = 1,
+         amazons_carrier = 40,
+         amazons_cassava_farmer = 3,
+         amazons_cocoa_farmer = 3,
+         amazons_stone_carver = 3,
+         amazons_geologist = 4,
+         amazons_woodcutter = 10,
+         amazons_liana_cutter = 4,
+         amazons_jungle_preserver = 5,
+         amazons_jungle_master = 3,
+         amazons_shipwright = 1,
+         amazons_hunter_gatherer = 3,
+         amazons_stonecutter = 2,
+         amazons_trainer = 3,
+         amazons_dressmaker = 1,
+         amazons_tapir = 5,
+         amazons_woodcutter_master = 3,
+}
+if campaign_data.payment then
+   soldiers = {
+      {{1,1,1,1}, 45},
+      {{2,2,1,2}, 55},
+      {{3,2,2,3}, 65},
+   }
+else
+   soldiers = {
+      {{1,1,1,1}, 70},
+      {{2,2,1,2}, 80},
+      {{3,2,2,3}, 90},
+   }
+end
+hq:set_soldiers(soldiers[difficulty][1], soldiers[difficulty][2])
+p4:place_building("amazons_fortification", map:get_field(55,190), false, true):set_soldiers(soldiers[difficulty][1], 12)
```

Comment:
the first 2 forttificatgions are placed to far from each other resulting in the AI nor beeing capable of connecing them. 
placing a 4th fortress at 47, 186 will solve this. Reducing each garrison to 9 would keep equal numbers

Reply:
Done. I kept the number of soldiers at 12 per site though – this scenario is easy ;) , so no reason not to make it a tiny bit harder